pkg/configuration/config.go

```diff
@@ -103,6 +103,26 @@ func (c *Config) GetVolumeFlavorName(flavorID string) (string, error) {
 	return "", fmt.Errorf("VolumeFalvorID %s not found", flavorID)
 }
 
+func (c *Config) GetStoragePoolName(storagePoolID string) (string, error) {
+	for _, sp := range c.AvailableResources.StoragePools {
```

Comment:
if id is not given, just return ""?  Or change the logic where the method gets called.

Reply:
see the other related comment

Reply:
never mind this comment either